openedx/core/djangoapps/content_tagging/api.py

```diff
@@ -175,6 +190,112 @@ def tag_content_object(
     return get_content_tags(object_key, taxonomy_id=taxonomy.id)
 
 
+def get_content_tags_for_object(
+    content_key: ContentKey,
+    include_children: bool,
+) -> tuple[TaggedContent, TaxonomyDict]:
```

Comment:
Hmm.. my recommended api method name wasn't great, sorry:  it's hard to tell the difference between it an `get_content_tags` and `oel_tagging.get_object_tags`.

Open to suggestions here!